gemrb/core/Scriptable/Actor.h

```diff
@@ -445,6 +445,17 @@ using vvcDict = std::multimap<ResRef, ScriptedAnimation*>;
 
 class GEM_EXPORT Actor : public Movable {
 public:
+	bool IsTraversable() const
+	{
+		if (!ValidTarget(GA_NO_DEAD | GA_NO_UNSCHEDULED)) {
+			return true;
+		}
+		if (ValidTarget(GA_ONLY_BUMPABLE)) {
+			return true;
+		}
+		return false;
+	}
```

Comment:
This looks unused and now should also go.

Reply:
Good catch, thanks.